hack/stabilization-changes.py

```diff
@@ -128,16 +141,21 @@ def stabilize_release(version, channel_name, channel_path, delay, errata, feeder
         else:
             yield '{}. {} {}'.format(subject, body, pull.html_url)
     else:
-        _LOGGER.info('  waiting: {} ({}){}'.format(version, version_delay, public_errata_message))
+        _LOGGER.info('  waiting: {} ({}){}'.format(version, merged_ago, public_errata_message))
         if waiting_notifications:
-            yield 'Recommend waiting to promote {} to {}; it was promoted the feeder {} by {} ({}, {}, {}){}'.format(
+            # Show either expected promotion time or when
+            time_message = None
+            if delay:
+                # Show expected date to promote
+                time_message = "Current promotion delay will be satisfied on {}".format(feeder_promotion['committer-time'] + delay)
+            else:
+                time_message = "Merged {}".format(merged_ago)
+            yield 'Recommend waiting to promote {} to {}; it was promoted the to feeder {} by {} {}){}'.format(
```

Comment:
I think you need `)  {}` at the end or some other way to get some whitespace in front of the text in `time_message`

Reply:
also `the to feeder` -> `to the feeder`

Reply:
Fixed in 34a8efcd0e9cb5c179a6d492c71fe33ee4a495bc